t/133-balancer.t

```diff
@@ -7,9 +7,9 @@ use Test::Nginx::Socket::Lua;
 #workers(2);
 #log_level('warn');
 
-repeat_each(2);
+repeat_each(1);
```

Comment:
@doujiang24 This is not a problem since grep_error_log_out supports array-typed values by which you can specify different results for each individual repeated request.